web/src/lightbox.ts

```diff
@@ -580,7 +580,7 @@ export function initialize(): void {
 
     $("#main_div, #compose .preview_content").on(
         "click",
-        ".message_inline_image:not(.message_inline_video) a",
+        ".message_inline_image:not(.message_inline_video) a, .message_inline_animated_image",
```

Comment:
Should this have a ` a` in the selector?

Reply:
No, then you won't be able to click on the play icon so as to open it in lightbox (other cases are anyways handled by `.message_inline_image:not(.message_inline_video) a`).

Reply:
OK